src/App.jsx

```diff
@@ -1,12 +1,19 @@
-import { Switch, Route } from 'react-router-dom';
+import React, { useContext } from 'react';
+import { Switch, Route, HashRouter } from 'react-router-dom';
 import { Header, Footer } from './components';
 import { Home, About, Contact, CityWeather } from './pages';
+import { DarkModeContext } from './contexts/darkMode';
+import { GlobalStyles } from './styles/globalStyles';
 import * as S from './App.styles';
 
 const App = () => {
+  const darkModeContext = useContext(DarkModeContext);
+  const { isDarkMode } = darkModeContext;
```

Comment:
lines can be merged into one

Reply:
Thanks, fixed.

Reply:
check other places where code also can be simplified

Reply:
Already fixed at both places where I am using it :)